Content.Client/_RMC14/Xenonids/Watch/XenoWatchBui.cs

```diff
@@ -65,19 +105,47 @@ private XenoWatchWindow EnsureWindow()
     {
         if (_window != null)
             return _window;
-
         _window = new XenoWatchWindow();
         _window.OnClose += Close;
         _window.SearchBar.OnTextChanged += OnSearchBarChanged;
+        _window.Tier0.OnToggled += OnTierToggled;
+        _window.Tier1.OnToggled += OnTierToggled;
+        _window.Tier2.OnToggled += OnTierToggled;
+        _window.Tier3.OnToggled += OnTierToggled;
+        _window.Tier4.OnToggled += OnTierToggled;
+        _window.Crusher.OnToggled += OnButtonToggled;
+        _window.Ravager.OnToggled += OnButtonToggled;
+        _window.Boiler.OnToggled += OnButtonToggled;
+        _window.Praetorian.OnToggled += OnButtonToggled;
+        _window.Spitter.OnToggled += OnButtonToggled;
+        _window.Carrier.OnToggled += OnButtonToggled;
+        _window.Hivelord.OnToggled += OnButtonToggled;
+        _window.Burrower.OnToggled += OnButtonToggled;
+        _window.Warrior.OnToggled += OnButtonToggled;
+        _window.Lurker.OnToggled += OnButtonToggled;
+        _window.Defender.OnToggled += OnButtonToggled;
+        _window.Drone.OnToggled += OnButtonToggled;
+        _window.Sentinel.OnToggled += OnButtonToggled;
+        _window.Runner.OnToggled += OnButtonToggled;
+        _window.Queen.OnToggled += OnButtonToggled;
+        _window.Parasite.OnToggled += OnButtonToggled;
+        _window.Lesser.OnToggled += OnButtonToggled;
+        _window.Larva.OnToggled += OnButtonToggled;
```

Comment:
This should be generated automatically from the list of xenos, not a control manually written for each one